src/lib.rs

```diff
@@ -331,9 +337,10 @@ impl ser::SerializeStruct for PythonDictSerializer<'_> {
     where
         T: Serialize,
     {
+        let py = PyNativeType::py(self.dict);
         Ok(self
             .dict
-            .set_item(key, pythonize(self.dict.py(), value)?)?)
+            .set_item(key, pythonize(py, value)?)?)
```

Comment:
Curious why this change was needed here? `dict.py()` is just a more concise form of `PyNativeType::py(self.dict)`